### DISCUSSION TITLE
Use for atmospheric correction

### DISCUSSION THREAD

**Discussion by dr226** [eradiate/eradiate#414] [Q&A]
Hello,
could this already be used to atmospherically correct common satellite data, e.g. Sentinel-2, or airborne acquisitions?
If so, what would be the steps? I'm familiar with MODTRAN, are there equivalent standard atmosphere profiles?

Thank you.

**Comment by leroyvn** [comment]
Hi @dr226, I'm not very familiar with atmospheric correction methods, but if your question is "Can I build an atmospheric correction lookup table with Eradiate", I think it's possible.

However, we don't have (yet) a pre-wired workflow for this. This means: setting up the right simulations, defining the dimensions on your lookup table, supplying the input data, etc. is mostly up to the user.

Answering the second part of your question: We do support the AFGL 1986 standard atmospheric profiles (the classic "US Standard", "Midlatitude summer", etc.). Our aerosol property library is currently limited, we are in the process of expanding it with basic classic models but we would also welcome contributions from users.
